### PR TITLE
Rename DbScripts_future to DbScripts_finalization

### PR DESCRIPTION
**Pull request by mimartin12**
## Objective
- Update our contributing docs with the recent renames of the `DbScripts_future` folders in https://github.com/bitwarden/server/pull/3192 to bring the folder names consistent with changes in https://github.com/bitwarden/contributing-docs/pull/166

### CODE REVIEW

**Review by differsthecat — COMMENTED** [review]
Not sure what the issue is as your changes look good, but the lint job is mad about something in `docs/contributing/database-migrations/index.md`.

**Comment by withinfocus** [comment]
This shouldn't merge until https://github.com/bitwarden/contributing-docs/pull/166 completes.

**Review by differsthecat — COMMENTED** [review]
@mimartin12 , This looks good now. Per @withinfocus 's comment, I'll approve once https://github.com/bitwarden/contributing-docs/pull/166 is merged!

**Review by differsthecat — APPROVED** [review]

**Pull request merged**
